addons/im_livechat/security/ir.model.access.csv

```diff
@@ -7,3 +7,13 @@ access_livechat_support_report_operator,im_livechat.report.operator,model_im_liv
 access_livechat_channel_rule,im_livechat.channel.rule,model_im_livechat_channel_rule,,1,0,0,0
 access_livechat_channel_rule_user,im_livechat.channel.rule,model_im_livechat_channel_rule,im_livechat_group_user,1,1,1,0
 access_livechat_channel_rule_manager,im_livechat.channel.rule,model_im_livechat_channel_rule,im_livechat_group_manager,1,1,1,1
+access_chatbot_script,chatbot.script,model_chatbot_script,,1,0,0,0
+access_chatbot_script_user,chatbot.script.user,model_chatbot_script,im_livechat_group_user,1,1,1,0
```

Comment:
I guess technically the scripts can't do anything more than what the support staff can do, but it still seems a bit iffy that any support staff can create and update scripts no?